modules/microshift-config-yaml.adoc

```diff
@@ -4,81 +4,43 @@
 
 :_content-type: CONCEPT
 [id="microshift-config-yaml_{context}"]
-= Using a YAML configuration file
+= Configuring {product-title}
 
-Configure your {product-title} using a YAML file. The values specified in your configuration file are used when you execute a command.
-// Q: can these values be overriden with CLI flags?
+{product-title} searches for a configuration file in the user-specific directory, `~/.microshift/config.yaml`, then the system-wide `/etc/microshift/config.yaml` directory. Create the configuring file, specifying any settings that should override the defaults, before starting MicroShift.
 
 [id="microshift-yaml-default_{context}"]
 == Default settings
 If you do not create a `config.yaml` file, the default values are used.
-// Q: what creates the default yaml?
-// Q: can default file be modified? (instead of making a new one)
 
 .Default YAML file example
 [source,yaml]
 ----
-cluster:
-  clusterCIDR: 10.42.0.0/16
-  serviceCIDR: 10.43.0.0/16
-  serviceNodePortRange: 30000-32767
-  dns: 10.43.0.10
-  domain: cluster.local
-  url: https://127.0.0.1:6443
-nodeIP: ""
-nodeName: ""
-logVLevel: 0
+dns:
+  baseDomain: microshift.example.com <1>
+network:
+  clusterNetwork:
+    - cidr: 10.42.0.0/16 <2>
+  serviceNetwork:
+    - 10.43.0.0/16 <3>
+  serviceNodePortRange: 30000-32767 <4>
+node:
+  hostnameOverride: "" <5>
+  nodeIP: "" <6>
+apiServer:
+  subjectAltNames: [] <7>
+debugging:
+  logLevel: "Normal" <8>
 ----
-
-[IMPORTANT]
+<1> Base domain of the cluster. All managed DNS records will be sub-domains of this base.
+<2> A block of IP addresses from which Pod IP addresses are allocated.
+<3> A block of virtual IP addresses for Kubernetes services.
+<4> The port range allowed for Kubernetes services of type NodePort.
+<5> The name of the node, defaults to hostname.
+<6> The IP address of the node, defaults to IP of the default route.
+<7> Subject Alternative Names for apiserver certificates.
+<8> Log verbosity. Valid values are Normal, Debug, Trace, or TraceAll.
+
+[NOTE]
 ====
-The configuration file must be located at the user-specific directory, `~/.microshift/config.yaml`, and the system-wide `/etc/microshift/config.yaml` directory. An existing user-specific `config.yaml` takes precedence.
+MicroShift only reads the configuration file on startup. Restart MicroShift after changing any configuration settings to have them take effect.
 ====
-//Q: can this be done with a command, or is this an instance where user goes into the directory and creates a text file?
```

Comment:
They just need to create the text file. This may be a case where less is more, because there are zillions of ways for them to create that file so just describing what it looks like and where it should be ought to be enough.

Reply:
Per the original engineering doc, this config file has to be in the directory listed in line 35. Should we be deleting line 35? That is the location where the file must ought to be, unless I am misunderstanding.

Reply:
That content was repeated from line 9, so I removed the duplication.